assets/node-exporter/daemonset.yaml

```diff
@@ -25,9 +25,11 @@ spec:
         - --no-collector.wifi
         - --collector.mountstats
         - --collector.cpu.info
+        - --collector.textfile.directory=.
```

Comment:
Could this be a specified directory instead of `.`? I think this way it should increase readability.

Reply:
That's the working dir, which simplifies you executing in, but yes, it could be injected in jsonnet.